assets/components/dough_theme/form/_form.scss

```diff
@@ -92,3 +92,27 @@
 .form__list {
   @include body(16, 24);
 }
+
+.form__custom-select {
+  width: 250px;
+  // This is copied from budget planner
+  height: 30px; // custom height
+  border: 1px solid; // $greyscale4; colour is a pacs variable
+  padding: 0 8px;
+  background: #fff right 10px center no-repeat;
+  background-image: image-url('dough/assets/images/select-arrow.png');
+  background-size: 15px 7px;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  // Pacs custom
+  // font-weight: 400;
```

Comment:
should this line be deleted?

Reply:
Yes, that is also defined in PACS and should be removed from here.